timestamp.md

```diff
@@ -63,6 +63,21 @@ A `TIMESTAMP` column supports values up to 12 bytes in width, but the total stor
 # Note that the first timestamp is UTC-05:00, which is the equivalent of EST.
 ~~~
 
+## Supported Casting & Conversion
+
+`TIMESTAMP` values can be cast to any of the following data types:
+
+Type | Details
+-----|--------
+`INT | Converts to number of seconds since the Unix epoch (Jan. 1, 1970)
```

Comment:
Missing closing back-ticks. But I want to get this in the release notes, so I'll merge and correct in a follow up.